packages/nextcloud/test/litmus_test.dart

```diff
@@ -0,0 +1,246 @@
+
+    group('basic', () {
+      test('options', () async {
+        final options = await client.webdav.options();
+        expect(options.capabilities, contains('1'));
+        expect(options.capabilities, contains('3'));
+        // Nextcloud only contains a fake plugin for Class 2 support: https://github.com/nextcloud/server/blob/master/apps/dav/lib/Connector/Sabre/FakeLockerPlugin.php
+        // It does not actually support locking and is only there for compatibility reasons.
+        expect(options.capabilities, isNot(contains('2')));
+      });
+
+      for (final (name, path) in [
+        ('put_get', 'res'),
+        ('put_get_utf8_segment', 'res-%e2%82%ac'),
+      ]) {
+        test(name, () async {
+          final content = Uint8List.fromList(utf8.encode('This is a test file'));
+
+          final response = await client.webdav.put(content, path);
+          expect(response.statusCode, 201);
+
+          final downloadedContent = await client.webdav.get(path);
+          expect(downloadedContent, equals(content));
+        });
+      }
+
+      test('put_no_parent', () async {
+        expect(
+          () => client.webdav.put(Uint8List(0), '409me/noparent.txt'),
+          // https://github.com/nextcloud/server/issues/39625
+          throwsA(predicate((final e) => (e! as DynamiteApiException).statusCode == 409)),
+        );
+      });
+
+      test('delete', () async {
+        await client.webdav.put(Uint8List(0), 'test.txt');
+
+        final response = await client.webdav.delete('test.txt');
+        expect(response.statusCode, 204);
+      });
+
+      test('delete_null', () async {
+        expect(
+          () => client.webdav.delete('test.txt'),
+          throwsA(predicate((final e) => (e! as DynamiteApiException).statusCode == 404)),
+        );
+      });
+
+      test('delete_fragment', () async {
+        // This test is not applicable because the fragment is already removed on the client side
+      });
+
+      test('mkcol', () async {
+        final response = await client.webdav.mkcol('test');
+        expect(response.statusCode, 201);
+      });
+
+      test('mkcol_again', () async {
+        await client.webdav.mkcol('test');
+
+        expect(
+          () => client.webdav.mkcol('test'),
+          throwsA(predicate((final e) => (e! as DynamiteApiException).statusCode == 405)),
+        );
+      });
+
+      test('delete_coll', () async {
+        var response = await client.webdav.mkcol('test');
+
+        response = await client.webdav.delete('test');
+        expect(response.statusCode, 204);
+      });
+
+      test('mkcol_no_parent', () async {
+        expect(
+          () => client.webdav.mkcol('409me/noparent'),
+          throwsA(predicate((final e) => (e! as DynamiteApiException).statusCode == 409)),
+        );
+      });
+
+      test('mkcol_with_body', () async {
+        // This test is not applicable because we only write valid request bodies
+      });
```

Comment:
Can we skip this?
Or even implement and skip it.

Same for the other empty tests. This would exclude them from visually from the other tests in my IDE and probably also remove the need to set up and destroy the docker container for them.

Reply:
Sure, I'd still like to keep the comment so later on we can see why the test doesn't exist